app/src/views/OperationalLearning/Stats/OperationalLearningMap/index.tsx

```diff
@@ -0,0 +1,267 @@
+                features,
+            };
+        },
+        [learningByCountry, countriesMap, countries],
+    );
+
+    const bluePointHaloCirclePaint: CirclePaint = useMemo(() => {
+        const countriesWithLearning = learningByCountry?.filter((value) => value.count > 0);
+
+        const maxScaleValue = countriesWithLearning && countriesWithLearning.length > 0
+            ? Math.max(
+                ...(countriesWithLearning
+                    .map((country) => country.count)),
+            )
+            : 0;
+
+        return {
+            'circle-opacity': 0.9,
+            'circle-color': [
+                'interpolate',
+                ['linear'],
+                ['number', ['get', 'learningCount']],
+                0,
+                LEARNING_COUNT_LOW_COLOR,
+                maxScaleValue,
+                LEARNING_COUNT_HIGH_COLOR,
+            ],
+            'circle-radius': [
+                'interpolate',
+                ['linear'],
+                ['zoom'],
+                3, 10,
+                8, 15,
+            ],
+        };
+    }, [learningByCountry]);
+
+    const handlePointClose = useCallback(() => {
+        setClickedPointProperties(undefined);
+    }, []);
+
+    const handlePointClick = useCallback(
+        (feature: mapboxgl.MapboxGeoJSONFeature, lngLat: mapboxgl.LngLatLike) => {
+            setClickedPointProperties({
+                feature: feature as unknown as ClickedPoint['feature'],
+                lngLat,
+            });
+            return true;
+        },
+        [setClickedPointProperties],
+    );
+
+    const maxLearning = useMemo(() => (
+        maxSafe(
+            learningByCountry?.map((value) => value.count),
+        )
+    ), [learningByCountry]);
+
+    const bounds = useMemo(
+        () => {
+            if (isNotDefined(learningByCountry)) {
+                return undefined;
+            }
+            const countryList = learningByCountry
+                .map((d) => countriesMap?.[d.country_id])
+                .filter(isDefined);
+            return getCountryListBoundingBox(countryList);
+        },
+        [countriesMap, learningByCountry],
+    );
+
+    return (
+        <Container
+            className={_cs(styles.operationLearningMap, className)}
+            footerClassName={styles.footer}
+            footerContent={(
+                <div className={styles.legend}>
+                    <div className={styles.legendLabel}>{strings.learningCount}</div>
+                    <div className={styles.legendContent}>
+                        <div
+                            className={styles.gradient}
+                            style={{ background: `linear-gradient(90deg, ${LEARNING_COUNT_LOW_COLOR}, ${LEARNING_COUNT_HIGH_COLOR})` }}
+                        />
+                        <div className={styles.labelList}>
+                            <NumberOutput
+                                value={0}
+                            />
```

Comment:
Any reason this is set as zero?

Reply:
it's the value for the lowest learning count. Defined a constant and used instead.